src/cmd_sym.ml

```diff
@@ -239,6 +239,15 @@ let cmd profiling debug unsafe optimize workers no_stop_at_failure no_values
       if no_stop_at_failure || not err then print_and_count_failures count tl
       else Ok count
   in
+  let sort_results deterministic_result_order res =
+    if deterministic_result_order then
+      let res = List.of_seq res in
```

Comment:
Not introduced in your PR but can we make `res` a `list` instead of a `seq` ? I would not be surprise if there was no reason for it to be a `seq`...

Reply:
There is very much reason to do. A Seq is lazy and hence allows outputting issues as soon as they are found, and not wait for the whole program to have been explored (which may not even terminate)

Reply:
Fine. I have a vague remembering of wanting to remove it at some point as it seemed not needed, but I'm fine with keeping it.